roles/kube-burner/files/metrics-aggregated.yaml

```diff
@@ -163,3 +163,10 @@ metrics:
 
   - query: sum(kube_node_status_condition{status="true"}) by (condition)
     metricName: nodeStatus
+
+  - query: sum(rate(container_fs_writes_total[5m])) by (device, container, node)
+    metricName: containerDiskUsage
+
+  - query: sum(rate(etcd_object_counts{}[5m])) by (resource)
```

Comment:
Adding > 0 at the end will reduce the number of timeseries:
Before:
![image](https://user-images.githubusercontent.com/4614641/96988403-95bb0600-1524-11eb-8a58-8b20983651f4.png)

After:
![image](https://user-images.githubusercontent.com/4614641/96988442-a4092200-1524-11eb-9e32-690fcf868468.png)

Reply:
ack, i'll add that.